### ISSUE TITLE
Empty file causes an error

### ISSUE THREAD

**Issue by Arcanemagus** [AtomLinter/linter-eslint#315]
When opening a completely empty file the following is triggered:
```
Error: EPERM: operation not permitted, write
    at Error (native)
    at Object.fs.writeSync (fs.js:663:20)
    at SyncWriteStream.write (fs.js:1981:6)
    at Console.log (console.js:36:16)
    at Object.module.exports.info (C:\Users\Landon Abney\Documents\GitHub-release\linter-eslint\node_modules\eslint\lib\logging.js:15:21)
    at Object.cli.execute (C:\Users\Landon Abney\Documents\GitHub-release\linter-eslint\node_modules\eslint\lib\cli.js:147:17)
    at C:\Users\Landon Abney\Documents\GitHub\linter-eslint\lib\worker.js:109:12
    at C:\Users\Landon Abney\Documents\GitHub\linter-eslint\lib\worker.js:88:18
    at C:\Users\Landon Abney\Documents\GitHub\linter-eslint\node_modules\zm-event-kit\dist\Emitter.js:58:16
    at Array.forEach (native)
```

**Comment by snowcxt** [comment]
+1
